esi_report.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# run ./esi_report.sh daily to get daily report
+# run ./esi_report.sh monthly to get last month's report
+# run ./esi_report.sh weekly to get last week's report
+# run ./esi_reports.sh plan to get future plan
+
+report_file=/tmp/esi_report
```

Comment:
It may make sense to set a report directory instead, and then have each report be a subdirectory within (/tmp/esi_report/monthly/, etc)